src/main/java/app/dto/project/ProjectRequestDto.java

```diff
@@ -0,0 +1,26 @@
+package app.dto.project;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.FutureOrPresent;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.time.LocalDate;
+
+public record ProjectRequestDto(
+        @NotBlank
```

Comment:
You can add messages here, like this
```suggestion
        @NotBlank(message = "Project name must be defined")
```